src/snowflake/snowpark/_internal/proto/ast.proto

```diff
@@ -1,5 +1,5 @@
 // N.B. This file is generated by `ir-dsl-c`. DO NOT EDIT!
-// Generated from `{git@github.com:snowflakedb/snowflake.git}/Snowpark/ast`.
+// Generated from `scratch/bazel/36bac8ee52e5d1559627454b12aa1b28/execroot/Snowflake/Snowpark/ast`.
```

Comment:
nit: is it possible to replace this to the commit from mono-repo

Reply:
Oooh, yeah. Let me do that separately.

Reply:
Let me clarify: I'll figure out a way to get back to the GIT location instead of the weird scratch directory. I think that's the right solution – I generally believe generated code should point to where the source of truth actually lives in case someone wants to make changes.